### PR TITLE
feat: update kubevirt tekton tasks documentation

### PR DESCRIPTION
**Pull request by ksimon1**
feat: update kubevirt tekton tasks documentation
This PR adds new task modify-windows-iso-file into KTT and updates names of example pipelines.
windows10-customize was renamed to windows-customize. 
Added documentation for new pipeline windows-efi-installer for win11/2k22

Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: docs/virtual_machines/tekton_tasks.md [hunk 1]
```diff
@@ -115,11 +115,18 @@ KubeVirt-specific Tekton tasks, which are focused on:
 #### Wait for Virtual Machine Instance Status
 - wait-for-vmi-status - Waits for a VMI to be running.
 
+#### Modify Windows iso
```
Comment (0xFelix) [inline docs/virtual_machines/tekton_tasks.md:118]: Can you explain why one should use the noprompt bootloader?

File: docs/virtual_machines/tekton_tasks.md [hunk 2]
```diff
@@ -115,11 +115,18 @@ KubeVirt-specific Tekton tasks, which are focused on:
...
+- modify-windows-iso-file - modifies windows iso (replaces prompt bootloader with no-promt bootloader) and replaces original iso 
+  in PVC with updated one.  This helps with automation of windows which requires EFI - the prompt bootloader will not continue with 
+  installation until some key is pressed. The non prompt bootloader will not require any key pres.
```
Comment (0xFelix) [inline docs/virtual_machines/tekton_tasks.md:121]: Typo: `pres`

File: docs/virtual_machines/tekton_tasks.md [hunk 3]
```diff
@@ -115,11 +115,18 @@ KubeVirt-specific Tekton tasks, which are focused on:
 #### Wait for Virtual Machine Instance Status
 - wait-for-vmi-status - Waits for a VMI to be running.
 
+#### Modify Windows iso
+- modify-windows-iso-file - modifies windows iso (replaces prompt bootloader with no-promt bootloader) and replaces original iso
```
Comment (0xFelix) [inline docs/virtual_machines/tekton_tasks.md:119]: Typo: `no-promt`

File: docs/virtual_machines/tekton_tasks.md [hunk 4]
```diff
@@ -115,11 +115,18 @@ KubeVirt-specific Tekton tasks, which are focused on:
 #### Wait for Virtual Machine Instance Status
 - wait-for-vmi-status - Waits for a VMI to be running.
 
+#### Modify Windows iso
+- modify-windows-iso-file - modifies windows iso (replaces prompt bootloader with no-promt bootloader) and replaces original iso 
+  in PVC with updated one.  This helps with automation of windows which requires EFI - the prompt bootloader will not continue with
```
Comment (0xFelix) [inline docs/virtual_machines/tekton_tasks.md:120]: `This helps with automated installation of Windows in EFI boot mode. By default Windows uses a prompt bootloader, which will not continue with the boot process until a key is pressed. By replacing it with the non-prompt bootloader no key press is required to boot into the Windows installer.`?

**Comment by ksimon1** [comment]
@phoracek, @dhiller would you please review this PR?

**Pull request merged**